_pages/faq.md

```diff
@@ -20,6 +20,13 @@ The graphic below illustrates the relationship between tests run by consumers an
 
 ![Illustration showing the lifecycle of an M-Lab test]({{ site.baseurl }}/images/about/lifecycle-of-mlab-test.png)
 
+## Where are M-Lab Servers hosted?
+{:.no_toc}
+
+M-Lab hosts our servers, which we call "pods", in Tier 1 data-centers, connected to the internet with links between 1-10 Gbps. In major metro areas, M-Lab strives to maintain multiple pods, each connected to as many distinct transit providers as possible.
```

Comment:
I think just a link here is fine. I don't think under About makes sense, but I'd want to revisit the site architecture notes that we have, where we had a plan for where it goes. Let's keep that as out of scope of this PR